cgmanifest.json

```diff
@@ -1,24 +1,24 @@
 {
-    "Registrations": [
-        {
-            "Component": {
-                "Type": "git",
-                "Other": {
-                    "RepositoryUrl": "https://github.com/bilke/cmake-modules",
-                    "CommitHash": "66fa2579d36f30cdab93d1dbd4c3d682a25ae5f3"
-                }
-            },
-            "DevelopmentDependency": false
```

Comment:
I changed `DevelopmentDependency` to true. From what I understand, false is the default and is meant for "true" dependencies. Our dependencies here are for code coverage cmake module and dev-time script, and it doesn't impact the SDK product directly.